test/StreamJsonRpc.Analyzer.Tests/Resources/EmptyInterface/IFoo.g.cs

```diff
@@ -0,0 +1,24 @@
+﻿// <auto-generated/>
+
+#nullable enable
+#pragma warning disable CS0436 // prefer local types to imported ones
+
+[global::StreamJsonRpc.Reflection.JsonRpcProxyMappingAttribute(typeof(StreamJsonRpc.Generated.IFoo_Proxy))]
+partial interface IFoo
+{
+}
+
+namespace StreamJsonRpc.Generated
+{
+	
```

Comment:
The formatting of these generated classes are a bit upsetting but I'm guessing there's a good reason for it?